### DISCUSSION TITLE
how to use quotes in vars and precondition section of taskfile

### DISCUSSION THREAD

**Discussion by raksac** [go-task/task#488] [Q&A]
This is how my taskfile looks like -

> version: '3'
>
>tasks:
>  dosomething:
>    cmds:
>      - runsomething {{.CLI_ARGS}}
>    vars:
>      DOCKER_GATEWAY:
>        sh: "$( docker network inspect bridge -f '{{range .IPAM.Config}}{{.Gateway}}{{end}}' )"
>    preconditions:
>      - test "$(systemctl is-active service)" = "active"
>      - test "$( docker container inspect -f '{{.State.Status}}' some )" = "running"

dryrun gives errors -
>$ task --dry dosomething -- some
`task: Command "$( docker network inspect bridge -f '' )" in taskvars file failed: exit status 2`

**Comment by ser-drephs** [comment]
Hey, Idk if you already figured it out after 2 years ^^  but as someone who came across the same issue - here's a possible solution.

Escape the `{{ }}` characters with `{{"{{"}}` and `{{"}}"}}`:

`- test "$( docker container inspect -f '{{"{{"}}.State.Status{{"}}"}}' some )" = "running"`
